news/2018-01-18-community-choice-2017.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which beatmaps were truly the greatest of the year. Cast your votes and help us figure out which maps from 2017 are truly the Community Favourites!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
+
+Everyone gets **10 votes per game mode** to cast for their favourite ranked maps of 2017 that they've played at least once before. The voting is super simple - find up to 10 of your favourite maps in any mode, shoot them a star via the contest listing, and you're done!
+
+The highest voted map for each gamemode will be declared the Community Favourite of 2017 and earn its mapper **a shiny new profile badge, and 6 months of osu!supporter** for their efforts.
```

Comment:
Change to -> "the Favourite of 2017, earning its mapper a shiny new profile badge and 6 months of"
uses 'and' less, and comma no longer seperates related clauses